scripts/startUserMigration.js

```diff
@@ -0,0 +1,44 @@
+const { program } = require('commander');
+const appPromise = require('../src/app');
+const { error } = require('../src/logger');
+const { schoolModel } = require('../src/services/school/model');
+
+program
+	.option('-s, --systemId <value>', 'id of the system, which will be added to the schools (required argument)')
+	.option(
+		'-S, --schoolIds <value...>',
+		'ids of the schools that should be migrated. If not given all schools with a official school number will be migrated.',
+	);
+
+program.parse();
+
+const options = program.opts();
+
+const startUserMigration = async () => {
+	const { systemId } = options;
+	const schoolsToMigrate = options.schoolIds;
+	let schools;
+	if (!schoolsToMigrate) {
+		schools = schoolModel.find({ officialSchoolNumber: { $exists: true } });
+	} else {
+		schools = schoolModel.find({ _id: { $in: schoolsToMigrate } });
+	}
+	for await (const school of schools) {
+		try {
+			school.ldapSchoolIdentifier = school.officialSchoolNumber;
```

Comment:
ldapSchoolIdentifier is a full dn, not only the school number (ou); take root path from system?